pom.xml

```diff
@@ -109,6 +113,12 @@
                 </configuration>
             </plugin>
         </plugins>
+        <resources>
```

Comment:
already in default package 
I think we don't have say it explicitly.

Reply:
i copied this part of pom from the sample project but i deleted this part